core/src/main/scala/org/allenai/common/Version.scala

```diff
@@ -59,10 +59,12 @@ object GitVersion {
   *
   * @param  git  the git version (commit information) of the build.
   * @param  artifactVersion  the version of the artifact in the build.
+  * @param  cacheKey a cacheKey of the project. changes on git commits to src of project and dependency changes
```

Comment:
Scaladoc follows javadoc style conventions - this should have the second sentence capitalized and ending with a period - `Changes ... and dependency changes.`